### PR TITLE
Fix rocsparse spitsv example residual check

### PR DESCRIPTION
**Pull request by MKKnorr**

### CODE REVIEW

File: Libraries/rocSPARSE/level_2/spitsv/main.cpp
Comment (zichguan-amd) [inline Libraries/rocSPARSE/level_2/spitsv/main.cpp]: CI is failing because newer changes in rocSPARSE changed the way iteration counts are done, resulting in off-by-one error. Otherwise LGTM.

Comment (MKKnorr) [reply Libraries/rocSPARSE/level_2/spitsv/main.cpp]: Thanks, the documentation is not very specific about this. Fixed it. The other CI failures are known issues.

**Review by zichguan-amd — APPROVED** [review]
All LGTM, but I'm not from the dev team so you might want to wait for someone else's approval.

**Pull request merged**
